src/segment-loader.js

```diff
@@ -908,6 +908,12 @@ export default class SegmentLoader extends videojs.EventTarget {
   trimBackBuffer_(segmentInfo) {
     const seekable = this.seekable_();
     const currentTime = this.currentTime_();
+    const targetDuration = this.playlist_.targetDuration || 10;
+
+    // Don't allow removing from the buffer within target duration of current time
```

Comment:
Is this from a different PR?

Reply:
With the changes to the safe live point, it created the problem where on some streams, we would be removing a portion of the GOP that currentTime was currently playing. This would cause chrome to remove the entire GOP, leaving currentTime outside of a buffered range, causing a playback stall.